src/app/portfolio/portfolio.component.html

```diff
@@ -16,6 +16,7 @@
         </div>
         <div class="h-full lg:w-80 flex flex-col justify-end items-end lg:mt-0">        
           <button app-action-button
+            [hidden]="(isClaimButtonVisible$ | async) === false"
```

Comment:
[Here](https://stackoverflow.com/a/51336717/4636976) are some opinions about hidden vs. *ngIf

Personally, I like *ngIf because it has the logic: "Show me this element if <something>", while hidden has: "This element is hidden if <something>". 
However, hidden is:
* more secure to obscure situations with undefined variables that are accessed inside the block (I don't like it as it silently fails and we continue to use incorrect code)
* a lot faster when the state rapidly changes as it doesn't manipulate the DOM directly, but just hides the still there elements, where *ngIf completely removes it from the DOM (for bigger blocks, *ngIf is better).

Use whatever you want, but be aware of their implications.

Reply:
I renamed `isClaimButtonVisible$` to `isClaimable$` so it's more general, not coupled just with the button visibility.